court_ball_game/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from court.models import Court
+from player.models import BallGame
+
+
+class CourtBallGame(models.Model):
+    court = models.ForeignKey(Court, on_delete=models.CASCADE)
+    ball_game = models.CharField(
+        max_length=100,
+        choices=BallGame.choices
+    )
```

Comment:
Add unique together constraint and associated test as in event player

Reply:
Done